### PR TITLE
Specify debug-report delivery

### PR DESCRIPTION
**Pull request by apasel422**
https://github.com/WICG/conversion-measurement-api/blob/main/EVENT.md#optional-extended-debugging-reports

#386

***
<a href="https://pr-preview.s3.amazonaws.com/apasel422/conversion-measurement-api/pull/404.html" title="Last updated on May 6, 2022, 5:11 PM UTC (ef218b0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/conversion-measurement-api/404/9db50a5...apasel422:ef218b0.html" title="Last updated on May 6, 2022, 5:11 PM UTC (ef218b0)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1107,13 +1110,37 @@ To <dfn>attempt to deliver a report</dfn> given an [=attribution report=] |repor
...
 A user agent MAY retry this algorithm in the event that there was an error.
 
+<h3 id="issue-debug-report-request">Issuing a debug report request</h3>
+
+To <dfn>attempt to deliver a debug report</dfn> given an [=attribution report=] |report|:
+
+1. Let |url| be the result of executing [=generate a report URL=] on |report| with
+    [=generate a report URL/isDebugReport=] set to true.
+1. Let |request| be the result of executing [=create a report request=] on |report|.
+1. [=Fetch=] |request|.
```
Comment (johnivdel) [inline index.bs:1138]: I think we still want to [=Queue a task=] to fetch request, we want it to be async.

File: index.bs [hunk 2]
```diff
@@ -1059,26 +1062,26 @@ whether a user is online, retries might be limited in number and subject to rand
 
 <h3 id="get-report-url">Get report request URL</h3>
 
-To <dfn>generate a report URL</dfn> for a given [=attribution report=] |report|, run the following steps:
+To <dfn>generate a report URL</dfn> for a given [=attribution report=] |report| and an optional
```
Comment (johnivdel) [inline index.bs:1065]:
```suggestion
To <dfn>generate a report URL</dfn> given a [=attribution report=] |report| and an optional
```

**Review by johnivdel — APPROVED** [review]
lgtm with comment

File: index.bs (hunk 1 above)
Comment (apasel422) [reply index.bs:1138]: But this algorithm was already called from a queued task.

Comment (johnivdel) [reply index.bs:1138]: Totally missed that, please ignore. (It'd be nice if there was someway to mark an algorithm async and require it be called in that manner)

**Pull request merged**